packages/hardhat/package-lock.json

```diff
@@ -0,0 +1,9448 @@
+{
+  "name": "@ethernauts/hardhat",
```

Comment:
can you commit without packagelock.json?

Reply:
Done